CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 
 # CHANGELOG
 
+## 0.20.0
+
+* Updated to Cruise Control 2.0.124 which fixed an issue with CPU utilization statistics for containers.
```

Comment:
Maybe we should just mention that we have enabled the CPU capacity goal?

Reply:
Good idea.